build.sbt

```diff
@@ -120,8 +121,10 @@ lazy val faciaJson_play26 = project.in(file("facia-json-play26"))
       Resolver.file("Local", file( Path.userHome.absolutePath + "/.ivy2/local"))(Resolver.ivyStylePatterns),
       "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
     ),
+    scalaVersion := scala211,
+    crossScalaVersions:= Seq(scala211, scala212),
+    releaseCrossBuild := true,
```

Comment:
Ah, so that's what we were missing yesterday!

Reply:
it's more than that. sbt 0.13 would disregard subproject `crossScalaVersions`

Reply:
Got it! thanks